index.js

```diff
@@ -48,15 +48,16 @@ function promiseFromStreams(streams) {
   })
 }
 
-module.exports = (streams, eventWriteKey, eventSinkHost) => {
+module.exports = (streams, eventWriteKey, eventSinkBaseUrl) => {
 
   if(!Array.isArray(streams)) { streams = [streams] }
 
   const serialize = ndjson.serialize(),
         compress = zlib.createGzip(),
         post = request.post({
-          url: `https://${eventSinkHost || 'event-sink.appuri.net'}/e?jsonConnectorApiKey=${eventWriteKey}`,
+          url: `${eventSinkBaseUrl || 'https://event-sink.appuri.net'}/e`,
```

Comment:
this changes the contract for the module. How about we add `https` if no scheme is there?